src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,71 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(foreignKey = @ForeignKey(name = "FK_movieSession_to_movie"))
+    private Movie movie;
+    @ManyToOne()
+    @JoinColumn(name = "cinemahall_id", foreignKey = @ForeignKey(name =
+            "FK_movieSession_to_cinemaHall"))
+    private CinemaHall cinemaHall;
+    @Column(name = "show_time")
+    private LocalDateTime showTime;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(final Long id) {
+        this.id = id;
+    }
+
+    public Movie getMovie() {
+        return movie;
+    }
+
+    public void setMovie(final Movie movie) {
+        this.movie = movie;
+    }
+
+    public CinemaHall getCinemaHall() {
+        return cinemaHall;
+    }
+
+    public void setCinemaHall(final CinemaHall cinemaHall) {
+        this.cinemaHall = cinemaHall;
+    }
+
+    public LocalDateTime getShowTime() {
+        return showTime;
+    }
+
+    public void setShowTime(final LocalDateTime showTime) {
+        this.showTime = showTime;
+    }
```

Comment:
Why you make parameters final?

Reply:
I can remove the word 'final'.
But I read the articles below and decided that if I do not change the argument inside the method, then it is useful to declare argument final.
Or is this a mistake?

https://stackoverflow.com/questions/500508/why-should-i-use-the-keyword-final-on-a-method-parameter-in-java
Never Reassign Arguments
As good programming practice (in any language), you should never re-assign a parameter/argument variable to an object other than the object passed by the calling method. In the examples above, one should never write the line arg = . Since humans make mistakes, and programmers are human, let’s ask the compiler to assist us. Mark every parameter/argument variable as 'final' so that the compiler may find and flag any such re-assignments.
-------------------------------------------------------------------------------

https://softwareengineering.stackexchange.com/questions/48413/in-java-should-i-use-final-for-parameters-and-locals-even-when-i-dont-have-t#:~:text=Java%20allows%20marking%20variables%20(fields,are%20meant%20to%20be%20immutable.

This is exactly why you should use the final keyword. You state that YOU know it'll never be re-assigned, but no one else knows that. Using final immediately disambiguates your code that tiny bit more.